src-cordova/config/dev/config.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <widget id="world.karrot.dev" version="9.3.0" xmlns="http://www.w3.org/ns/widgets" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
     <name>Karrot Dev</name>
-    <description>Modern website for organization of foodsaving groups worldwide</description>
+    <description>Independent group organization</description>
     <author email="karrot@foodsaving.world" href="https://foodsaving.world">
```

Comment:
I think info@karrot.world is more suitable. But I would not know where to put a link to the authorship actually, maybe point towards https://community.foodsaving.world/people